### DISCUSSION TITLE
Places: Update Location Infos Only 📍

### DISCUSSION THREAD

**Discussion by web2wire** [photoprism/photoprism#1721] [General]
Hi,

I have just set up PhotoPrism on a Flatcar Linux VM and so far it seems to be happy. I left it indexing my photos for the last 24 hours or so but overnight there was an internet outage which meant that a couple hundred GeoIP lookups failed.

Is there a way (via UI or CLI) to rerun these lookups without a full rescan?

Thanks

Phil

**Comment by lastzero** [comment]
Thanks for reporting this! Thought about adding a places CLI comment many times but never had a chance due to all the other feature requests... Hopefully soon.

**Reply by web2wire** [reply]
Thanks for the quick reply. In the meantime am I correct in assuming it's a full rescan that's needed to update the geo info for those photos that is got skipped for?

**Reply by lastzero** [reply]
Yes. Sorry to have no better news...

**Reply by lastzero** [reply]
Next Development Preview comes with a `photoprism places update` command: https://docs.photoprism.app/getting-started/updates/#development-preview

Note it's brand new and pretty much untested outside my development environment. You're welcome to report bugs if you find any.
